tests/api/admin/controller/test_work_editor.py

```diff
@@ -1003,7 +1003,7 @@ def test_process_cover_image(self):
             )
             / len(image_histogram)
         )
-        assert root_mean_square < 0.01
+        assert root_mean_square < 10
```

Comment:
@io7m Would you add a comment here about how you arrived at this RMS threshold? Also, it's used a couple of times, so it could be helpful to add a descriptive variable and put the explanation on that.